### DISCUSSION TITLE
Welcome to transbank-sdk-php Discussions!

### DISCUSSION THREAD

**Discussion by gdespirito** [TransbankDevelopers/transbank-sdk-php#158] [General]
## 👋 Welcome!
  We’re using Discussions as a place to connect with other members of our community. We hope that you:
  * Ask questions you’re wondering about.
  * Share ideas.
  * Engage with other community members.
  * Welcome others and are open-minded. Remember that this is a community we
  build together 💪.

  To get started, comment below with an introduction of yourself and tell us about what you do with this community.

**Comment by sunitha49** [comment]
Hi,

We have installed web pay magento2 modules (TransbankDevelopers/transbank-plugin-magento2-webpay) ,  transbank-sdk-php compared to SDK versions 1.x and 2.x is completely different, but after installing 2.x, it's asking the 1.x files (init.php, configuration.php), may be some files are missing in 2.x. Please respond as soon as possible.

**Reply by gdespirito** [reply]
Hi, i see this is not an issue of the SDK. In that case, please create an issue or communicate with our team in slack: https://join-transbankdevelopers-slack.herokuapp.com/

**Comment by vientito** [comment]
Hello, 

I developed a software that integrated the libraries of transbank webpay codes back in 2018. Recently visited transbank site and noticed commercial code has been changed and coming to this site i realized the libraries seem to have taken a major facelift. I wonder if most of servers are still backward compatible to codes using the then published libraries from 2018?
